src/lib/wallet/IndyWallet.ts

```diff
@@ -68,6 +63,20 @@ export class IndyWallet implements Wallet {
     return this.indy.createAndStoreMyDid(this.wh, didConfig || {});
   }
 
+  async createCredDef(
```

Comment:
Also, why is this included in the wallet? I think we could do it in the schema service. Seems like it doesn't belong here

Reply:
I have the same problem with it, but Indy function inside has this signature `indy.issuerCreateAndStoreCredentialDef(this.wh, issuerDid, schema, tag, signatureType, config)` which requires wallet handle (`wh`) and I didn't want to expose it directly. Maybe, what we could do is to add `getWalletHandle` method.

Reply:
I would keep it as it is right now. I don't like either of the solutions. Maybe, when we'll be implementing integration with other ledgers we would know more how to approach it correctly